src/server/chromium/crBrowser.ts

```diff
@@ -50,6 +50,9 @@ export class CRBrowser extends Browser {
     const browser = new CRBrowser(connection, options);
     browser._devtools = devtools;
     const session = connection.rootSession;
+    if ((options as any).__testHookOnConnectToBrowser)
```

Comment:
Doesn't existing `__testHookBeforeCreateBrowser` work?

Reply:
I wanted to wait after the websocket connection is established. What if the DevTools initialization does something async before reporting the first target?

Reply:
Sure.